e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -132,3 +134,155 @@ test('change camera, show planes', async ({ page, context }) => {
+  await u.waitForCmdReceive('extrude')
+  await page.waitForTimeout(1000)
+  await u.clearAndCloseDebugPanel()
+
+  await page.getByRole('button', { name: 'KittyCAD Modeling App' }).click()
+
+  const doExport = async (output: Models['OutputFormat_type']) => {
+    await page.getByRole('button', { name: 'Export Model' }).click()
+
+    // click the select box with name="type"
+    const exportSelect = page.getByTestId('export-type')
+    await exportSelect.selectOption({ label: output.type })
+
+    if ('storage' in output) {
+      const storageSelect = page.getByTestId('export-storage')
+      await storageSelect.selectOption({ label: output.storage })
+    }
+
+    const downloadPromise = page.waitForEvent('download')
+    // await page.getByText('Download file').click();
+    await page.getByRole('button', { name: 'Export', exact: true }).click()
+    const download = await downloadPromise
+
+    const downloadLocation = `./e2e/playwright/export-snapshots/${
+      output.type
+    }-${'storage' in output ? output.storage : ''}.${output.type}`
+    await download.saveAs(downloadLocation)
+    if (output.type === 'step') {
+      // s/[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]+[0-9]+[0-9]\+[0-9]{2}:[0-9]{2}/1970-01-01T00:00:00.0+00:00/g
+      // replace date in download file
+      const fileContents = await fsp.readFile(downloadLocation, 'utf-8')
+      const newFileContents = fileContents.replace(
+        /[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]+[0-9]+[0-9]\+[0-9]{2}:[0-9]{2}/g,
+        '1970-01-01T00:00:00.0+00:00'
+      )
+      await fsp.writeFile(downloadLocation, newFileContents)
+    }
+  }
+  const axisDirectionPair: Models['AxisDirectionPair_type'] = {
+    axis: 'z',
+    direction: 'positive',
+  }
+  const sysType: Models['System_type'] = {
+    forward: axisDirectionPair,
+    up: axisDirectionPair,
+  }
+  await doExport({
+    type: 'step',
+    coords: sysType,
+  })
+  await doExport({
+    type: 'gltf',
+    storage: 'embedded',
+    presentation: 'pretty',
+  })
+  await doExport({
+    type: 'gltf',
+    storage: 'binary',
+    presentation: 'pretty',
+  })
+  await doExport({
+    type: 'gltf',
+    storage: 'standard',
+    presentation: 'pretty',
+  })
+  await doExport({
+    // obj seems to be a little flaky, times out tests sometimes
+    type: 'obj',
+    coords: sysType,
+    units: 'in',
+  })
+
+  // the following exports are failing
+  // await doExport({
+  //   type: 'ply',
+  //   storage: 'ascii',
+  //   coords: sysType,
+  // })
+  // await doExport({
+  //   type: 'ply',
+  //   storage: 'binary' as unknown as 'binary_little_endian',
+  //   coords: sysType,
+  // })
+  // await doExport({
+  //   type: 'stl',
+  //   storage: 'ascii',
+  //   coords: sysType,
+  //   units: 'in',
+  // })
+  // await doExport({
+  //   type: 'stl',
+  //   storage: 'binary',
+  //   coords: sysType,
+  //   units: 'in',
+  // })
```

Comment:
@jessfraz, not sure who else to ping,

looks like this has already caught some bugs. stl and ply exports fail and is reproducible outside of playwright.

I created https://github.com/KittyCAD/modeling-app/issues/1140